lib/mpl_toolkits/basemap/test.py

```diff
@@ -96,6 +96,59 @@ def test_no_cyc2(self):
         assert (grid==gridout).all()
 
 
+class TestShiftdata(TestCase):
+
+    def _get_2d_lons(self, lons1d):
+        """
+        Generate a 2d grid
+        """
+        lats = [10, ] * len(lons1d)
+        return np.meshgrid(lons1d, lats)[0]
+
+    def test_2_points_should_work(self):
+        """
+        Shiftdata should work with 2 points
+        """
+        bm = Basemap(llcrnrlon=0, llcrnrlat=-80, urcrnrlon=360, urcrnrlat=80, projection='mill')
+
+        lons_expected = [10, 15, 20]
+        lonsout = bm.shiftdata(lons_expected.copy())
```

Comment:
In python3 they do)) Sorry I am using python3, do not have python2 to check this... There is a method that picks them all, are you going to register these as well? I am developing and running tests in Pycharm, which allows to just select tests to run)) Sorry about the omissions..

Cheers